posts/2024-01-24_wyg-zooma/README.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Who You Gno – On the Record with Antoine Breuil"
+publication_date: 2024-01-11T00:00:00Z
+slug: wyg-zooma
+tags: [whoyougno, teritori, community, interview]
+authors: [christina]
+---
+
+Antoine Breuil, ‘zôÖma,’ is the co-founder of [Teritori](https://app.teritori.com/), an active Gno.land contributor and grantee that’s building key modules and tooling for Gno. A firm believer in equal opportunities, free and decentralized access to information, and helping fellow humans, zôÖma is fascinated by human behavior and how we organize ourselves, holding an avant-garde social experiment five years ago with Teritori co-founder ‘Pop.’ "La Suite du Monde" drew people across France to a small village in the countryside to create a shared community and society—with farmland, accommodation, and tools for common use.
+
+The goal was to form an in-real-life DAO whose members shared common goals and interests—using blockchain technology with a token to pay for goods and services and vote on governance matters. While many people participated and were enthusiastic about creating a shared society, zôÖma admits the experiment was early: no one was interested in interacting with the tech. “90% of people, rural or not, found it too complicated,” he admits. “We were a bit naive, but it was fascinating nonetheless.”
+
+## A Fascination for Human Behavior
+
+zôÖma has been an ardent student of human behavior since childhood. His parents taught him early on the value of philanthropy and working with people in need. He’s set up several joint liability companies, non-profits, and NGOs to experiment with finding new and better ways to organize society, and one of the things he loves most about web3 is its “experimental” nature. He’s encouraged by how far the industry has come since he received his first bitcoins in partial payment for a website in 2014. “That turned out to be a really expensive website for my customer,” he laughs. He never expected such broad adoption of Bitcoin and a technology that “inspired a whole generation of engineers to experiment with new things.”
+
+Like most creative types, zôÖma is used to spinning many plates in the air, overseeing La Suite du Monde while working as a freelance designer, front-end dev, and Artistic Director for an independent French record label. “Before entering the world of engineers, I founded and managed a collective for 12 years, which brought together artists from all disciplines, hackers, designers, tinkerers, to build some interesting projects.” La Suite de Monde allowed him to explore his passion for finding new approaches to social coordination first-hand. “I explored very radical things,” he says, “like the notion of “accepted by default” where anyone could use the collective budget by expressing their desire to do so three times. I wouldn’t recommend this,” he laughs, “but the experiments were fascinating and still serve me today in my work.”
```

Comment:
> "..the notion of “accepted by default” where anyone could use the collective budget by expressing their desire to do so three times."

I am not sure what this means without further context. Does this mean that after 3 times of expressing interest in something that someone's idea is accepted without debate?

Reply:
Yes